README.md

```diff
@@ -1,3 +1,27 @@
-# JaneliaDataStandards
-A central hub for Janelia's bioimaging schemas, standards, and data formats.
-This project is at a very preliminary stage.
+# Janelia Data Standards
+
+## Purpose
+
+The Janelia Data Standards group was formed by bioimaging developers who have encountered specific, practical bioimaging data dilemmas that international community standards don’t yet cover. When cutting-edge experiments produce new kinds of data or metadata, developers may be forced to modify existing file formats, or occasionally create new ones. For example, it is often unclear how to store and describe high dimensional images, image tiles that form parts of a whole, or images accompanied by large ancillary data sets. Even though developers may be making reasonable choices on their own, a chaotic landscape can result if they do not communicate their choices, and encourage others to store similar data in a similar way.
+
+**This website is Janelia’s bioimaging developers’ manifesto.** It is a collection of essays written by developers, for developers, on the advanced technical quandaries they’ve encountered. It records the choices Janelia’s developers have made when encountering exotic data dilemmas, so that those encountering similar dilemmas can make consistent choices. Janelia is excited about international standardization efforts, particularly OME-NGFF. Janelia is contributing to OME-NGFF and is rooting for its success. However, such efforts are not enough, since Janelia's experiments will always be one step ahead of community conventions. This manifesto is meant to fill that gap.
```

Comment:
> "...since Janelia's experiments will always be one step ahead of community conventions"

Might be a little presumptuous.

Reply:
How about: 

>"...since Janelia's data needs may often outpace or even diverge from contemporary community standards."

Reply:
"manifesto" has political connotations. Could you explain this choice of word?

Reply:
@rhoadesScholar I'm fine with your suggestion, or:

> "...since research experiments will always be one step ahead of community conventions."

I wasn't trying to say Janelia is special, just that research, by definition, pushes boundaries.

@mkitti I like 'manifesto' because it implies the work is driven by ideological conviction, and that the principles are meant to be authoritative. I'm open to alternatives, but I don't love the ones that come to mind:
**'blog'** - Feels wishy-washy, like these are just passing observations.
**'guidelines / conventions / standards / style guide'** - These terms feel confusing because they suggest a list of rules, rather than a collection of stories.
But as I say, I'm willing to be persuaded otherwise.

Reply:
```suggestion
**This website is Janelia’s bioimaging developers’ manifesto.** It is a collection of essays written by developers, for developers, on the advanced technical challenges they’ve encountered. It records the choices Janelia’s developers have made when encountering exotic data, so that those encountering similar situations can make consistent choices. Janelia is excited about international standardization efforts, particularly OME-NGFF. Janelia is contributing to OME-NGFF and is rooting for its success. However, such efforts are not enough, since Janelia's experiments will always be one step ahead of community conventions. This manifesto is meant to fill that gap.
```
The word dilemma (+ synonyms that I was not familiar with) is used a lot. I would probably substitute this with a more straightforward word, so that this manifesto is applicable to a broad set of cases. Being in a dilemma is very subjective.